Backend/Remora.Discord.Gateway/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -58,12 +58,27 @@ public static IServiceCollection AddDiscordGateway
         Action<IHttpClientBuilder>? buildClient = null
     )
     {
-        serviceCollection
-            .AddDiscordRest
-            (
-                s => (tokenFactory(s), DiscordTokenType.Bot),
-                buildClient
-            );
+        serviceCollection.AddSingleton<IAsyncTokenStore>(ctx => new StaticTokenStore(
```

Comment:
Formatting. If you need to break lines, use allman style (same as with braces) for parens as well.

Reply:
Formatted appropriately